src/platform/imx8ulp/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+add_subdirectory(lib)
```

Comment:
On a second thought please split this into two patches:
* one that adds Xtensa specific headers
* the rest